### PR TITLE
Feature/gov 60 show user

### PR DESCRIPTION
**Pull request by aleciafb**
User show page:
- Fetch item on route change
- Actions for item fetch and receive
- Reducer adjustment to make use of entity id instead of index nr

### CODE REVIEW

File: src/components/User/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const User = ({ user, isLoading }) => (
+    <div>
+        {isLoading ? (
+            <p>Loading...</p>
+        ) : (
+            <div>
```
Comment (jwfwessels) [inline src/components/User/index.js]: 🐒 
is it realistic to assume there is data if the endpoint isnt loading? what about if there was an error or something?

maybe `isLoading || !user`?

File: src/actions/lists.js [hunk 2]
```diff
@@ -9,12 +11,22 @@ export const fetch = entity => () => ({
     type: `${FETCH}/${entity}`
 });
 
+export const fetchItem = entity => () => ({
```
Comment (jwfwessels) [inline src/actions/lists.js]: 🐒 
Do you think it makes sense to add this logic to the lists action factory thing 🤔 
I cant think why not im just wondering about it now.

File: src/components/Users/index.js [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
-                    <th>Email</th>
-                </tr>
-                {userIds.map(userId => (
-                    <tr key={users[userId].id}>
-                        <td>
-                            {users[userId].first_name} {users[userId].last_name}
-                        </td>
-                        <td>{users[userId].username}</td>
-                        <td>{users[userId].email}</td>
+        {isLoading ? (
```
Comment (jwfwessels) [inline src/components/Users/index.js]: 🐒 
same as above

File: src/components/User/index.js (hunk 1 above)
Comment (aleciafb) [reply src/components/User/index.js]: Valid point. I'll change it to 
```
{isLoading && <p>Loading</p>}
{user && <div>...User display code...</div>}

```

Error handling an notifications I'll handle when doing the errorboundry

File: src/actions/lists.js (hunk 2 above)
Comment (aleciafb) [reply src/actions/lists.js]: @jwfwessels Actions factory?

**Pull request merged**